azure/template.json

```diff
@@ -93,6 +93,13 @@
             "metadata": {
                 "description": "Base URL of new FIND"
             }
+        },
+        "featureRedirectToRails": {
+            "type": "bool",
+            "defaultValue": {},
```

Comment:
default value should be either true or false (without quotes or brackets) for boolean type variables

Reply:
Default would be `false`